### PR TITLE
HW-2

### PR DESCRIPTION
**Pull request by Pavlo-Yes**

### CODE REVIEW

File: src/components/footer/Footer.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ export function Footer(props) {
       <div className="may-footer-content">
       <div className="may-footer-text">Some text about authors</div>
       <div className="may-footer-social-media-wrapper">
-        <img className="may-footer-social-media-icon" src={socialMediaIcons[0].src} alt={socialMediaIcons[0].alt}/>
-        <img className="may-footer-social-media-icon" src={socialMediaIcons[1].src} alt={socialMediaIcons[1].alt}/>
-        <img className="may-footer-social-media-icon" src={socialMediaIcons[2].src} alt={socialMediaIcons[2].alt}/>
+          {socialMediaIcons.map(value =>
+              <img className="may-footer-social-media-icon" src={value.src} alt={value.alt}/>
```
Comment (AlyonaPianykh) [inline src/components/footer/Footer.js:14]: `key` props is missing. Don't forget to add it =)

File: src/components/header/HeaderFromLecture.js [hunk 2]
```diff
@@ -14,15 +14,11 @@ export const Header = (props) => {
...
-          <a href={links[0].url} className="may-header-links-wrapper-link nav-link">{links[0].name}</a>
-        </div>
-        <div className="nav-item">
-          <a href={links[1].url} className="may-header-links-wrapper-link nav-link">{links[1].name}</a>
-        </div>
-        <div className="nav-item">
-          <a href={links[2].url} className="may-header-links-wrapper-link nav-link">{links[2].name}</a>
-        </div>
+          {links.map(value =>
+              <div className="nav-item">
```
Comment (AlyonaPianykh) [inline src/components/header/HeaderFromLecture.js:18]: same here

**Comment by AlyonaPianykh** [comment]
Review done

**Comment by Pavlo-Yes** [comment]
done, add key in footer,header.

**Pull request closed without merging**